autofs-5.1.4 - add an example fedfs master map entry to the installed master map

From: Ian Kent <raven@themaw.net>

Add an example of using the fedfs program map (to facilitate access
to fedfs domain mounts) to the installed master map.

Signed-off-by: Ian Kent <raven@themaw.net>
---
 CHANGELOG           |    1 +
 samples/auto.master |    6 ++++++
 2 files changed, 7 insertions(+)

diff --git a/CHANGELOG b/CHANGELOG
index bbe8019e..f771a433 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -18,6 +18,7 @@ xx/xx/2018 autofs-5.1.5
 - add mount.fedfs.c.
 - add fedfs-map-nfs4.c.
 - add conditional inclusion of fedfs binaries.
+- add an example fedfs master map entry to the installed master map.
 
 19/12/2017 autofs-5.1.4
 - fix spec file url.
diff --git a/samples/auto.master b/samples/auto.master
index 0f2c8ab0..9a9968a3 100644
--- a/samples/auto.master
+++ b/samples/auto.master
@@ -17,6 +17,12 @@
 #
 +dir:/etc/auto.master.d
 #
+# If you have fedfs set up and the related binaries, either
+# built as part of autofs or installed from another package,
+# uncomment this line to use the fedfs program map to access
+# your fedfs mounts.
+#/nfs4  /usr/sbin/fedfs-map-nfs4 nobind
+#
 # Include central master map if it can be found using
 # nsswitch sources.
 #
